WordPress/Classes/ViewRelated/Post/PostSettings/PostSettingsViewModel.swift

```diff
@@ -113,6 +114,7 @@ final class PostSettingsViewModel: NSObject, ObservableObject {
     }
 
     private let originalSettings: PostSettings
+    private let moc: NSManagedObjectContext
```

Comment:
This variable is not used.

Reply:
Fixed. It was moved to the service and I forgot to remove it.